index.d.ts

```diff
@@ -96,4 +96,22 @@ declare module "react-nice-dates" {
   export function DateRangePickerCalendar(
     props: DateRangePickerCalendarProps
   ): JSX.Element;
+  
+  export function useDateInput({
+    date,
+    format,
+    locale,
+    minimumDate,
+    maximumDate,
+    onDateChange,
+    validate
+  }: {
+    date?: Date,
+    format?: string,
+    locale: Locale,
+    minimumDate?: Date,
+    maximumDate?: Date,
+    onDateChange?: (date: Date) => void,
```

Comment:
`onDateChange` should be required as well (since if you don't pass it there will be an error when the hook tries to call it after a change).

Reply:
onDateChange is now marked as required